openqa-label-known-issues

```diff
@@ -142,6 +142,10 @@ investigate_issue() {
     # see https://progress.opensuse.org/issues/69178
     echo "$reason" >> "$out"
     if [[ "$curl_output" != "200" ]] && [[ "$curl_output" != "301" ]]; then
+        # Assume autoinst-log.txt still not found but process job anyway due to reason var
+        if [[ -n $reason ]] && [[ "$curl_output" = "404" ]] && [[ $reason != null ]]; then
+            label_on_issues_from_issue_tracker "$id"
```

Comment:
I'm not really familiar with this script at this point but don't we need the early `return` here as well? And if we have it, we still somehow need to ensure `label_on_issues_without_tickets` and `handle_unreviewed` are called in case the other function doesn't find anything. So *maybe* this need a little bit more restructuring.

Reply:
There is no `return` because I believe we still need to go through `handle_unreachable_or_no_log` which will perform a return anyway. And IIUC the second part is covered in this case, as it continues to follow the workflow as before. right?

Reply:
This question should be clarified via https://progress.opensuse.org/issues/164296#note-13.